src/CMakeLists.txt

```diff
@@ -526,6 +526,12 @@ add_custom_target(make_stdlib ALL
   COMMAND $(MAKE) -f ${CMAKE_BINARY_DIR}/stdlib.make Init Lean
   VERBATIM)
 
+# if we have LLVM enabled, then build `lean.h.bc` which has the LLVM bitcode
```

Comment:
@Kha , could you explain to me why moving this code to `src/CMakeLists.txt` fixes the build on `Linux Release`? To summarize the change, before this commit, [the build breaks with](https://github.com/leanprover/lean4/actions/runs/5608926936/jobs/10261983565#step:9:1343)

```
lean_llvm_create_memory_buffer_with_contents_of_file 
file:/home/runner/work/lean4/lean4/build/stage0/../stage1/lib/lean/lean.h.bc
ERROR:      No such file or directory
```

and after this commit, the linux release build succeeds! I don't understand the build-time control flow, and why moving the file here helps.

Reply:
The important part is not the code move but replacing `lean` with `make_stdlib`. Which is the target that otherwise generates the error you quoted.